### DISCUSSION TITLE
Lab Assignment #2

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#9] [Assignments]
### Due 7/9

Please use this thread to post your [Lab Assignment #2](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_02/Materials/lab_assignment.md)'s GitHub pages URL.

For example:
https://albertkun.github.io/211A-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by cynthinee** [comment]
https://cynthinee.github.io/2021A-191A-Lab-Assignments/week2/

**Comment by edmondywen** [comment]
https://edmondywen.github.io/asiaam-191a/week2/index.html

**Comment by abi-200** [comment]
https://abi-200.github.io/191a-labs/week2/

(scroll down for buttons!)

**Comment by luigimagdamit** [comment]
https://luigimagdamitucla.github.io/211A-191A-Lab-Assignments/week2/

**Comment by hrfrahn** [comment]
https://hrfrahn.github.io/191A-Labs/week2/

**Comment by perezsara59** [comment]
https://perezsara59.github.io/211A-191A-Assignments/week2/

**Comment by emilysuh** [comment]
https://emilysuh.github.io/211A-191A-Lab-Assignment/week2/index.html

**Comment by eandrusenki** [comment]
https://eandrusenki.github.io/AsAmS-191A-Assignments/Week2/

**Comment by ramirezirvin** [comment]
https://ramirezirvin.github.io/191A-lab-assingments/week2/

**Comment by ronnie-blanche** [comment]
http://10.1.10.179:61302/Week%202/
